Source/Graphics/OpenGL/GLRenderer2D.cpp

```diff
@@ -327,7 +335,7 @@ void GLRenderer2D::prep(Texture* texture, const Color &color, float opacity, con
 		{
 			if(m_WantedAttributes[i] != m_EnabledAttributes[i])
 			{
-				disableAttribute(i);
+				glDisableVertexAttribArray(i);
```

Comment:
Want to make sure this is disabled here so I'm calling it directly, regardless of whether or not it was previously enabled.